.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - main
+      - development
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  FORCE_COLOR: 3
+
+jobs:
+  # pre-commit:
+  #   name: Format
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #       with:
+  #         fetch-depth: 0
+  #     - uses: actions/setup-python@v4
+  #       with:
+  #         python-version: "3.x"
+  #     - uses: pre-commit/action@v3.0.0
+  #       with:
+  #         extra_args: --hook-stage manual --all-files
+  #     - name: Run PyLint
+  #       run: |
+  #         echo "::add-matcher::$GITHUB_WORKSPACE/.github/matchers/pylint.json"
+  #         pipx run nox -s pylint
```

Comment:
I've currently disabled that. It is supposed to run, but that means the CI will break since it will do a lot of linting for code style. I didn't wanna make any code changes in terms of styling, so I disabled it. @uwcdc could you remove these commented lines?

Reply:
The lines have been removed.